src/components/StylesFromLiteralsService.js

```diff
@@ -219,47 +228,69 @@ goog.provide('ga_stylesfromliterals_service');
         return this.defaultStyle;
       };
 
+      OlStyleForPropertyValue.prototype.setOlText_ = function(olStyle,
+        labelProperty, labelTemplate, properties) {
+        var text;
+        properties = properties || [];
+        if (labelProperty) {
+          text = properties[labelProperty];
+          if (text !== undefined && text !== null) {
+            text = text.toString();
+          }
+        } else if (labelTemplate) {
+          text = labelTemplate;
+          angular.forEach(properties, function(prop, k) {
+            if (prop !== undefined && prop !== null) {
+              text = text.replace('${' + k + '}', prop.toString());
```

Comment:
prop is already a string I guess, no nee to convert it.

Reply:
prop is the value not the key

Reply:
Whatever prop is,  replace will work fine without the `toString`

Reply:
yes but I'd rather have the template value instead.

Reply:
What I meant is the `replace` function does the `toString()` for you, so no need to use it.

Reply:
yes ok